text.txt

```diff
@@ -0,0 +1,212 @@
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div>
+                <div>
+                    <h2>About me</h2>
+                    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                    <button>Get my resume</button>
```

Comment:
please let us know the type of button and the tag. so that the website can understand the function of the button